### ISSUE TITLE
lightbar is not automatically controlled by carma (first noticed on trucks)

### ISSUE THREAD

**Issue by MishkaMN** [usdot-fhwa-stol/carma-platform#937]
### Types of Issue

- [X] Anomaly report (something appears to not work correctly)
- [ ] Enhancement request (describe the enhancement being requested)
- [ ] Other (please ensure the description clarifies why the issue doesn’t fall into either of the above categories)

### Descriptive summary

It was brought to our attention that the lightbar does not correctly reflect the automation states of the carma on trucks.
Involved components are lightbar_driver and lightbar_manager.
Lightbar_manager was never tested on passenger cars, but only was on mock drivers, so potential error could be there.

This does not affect functional operation of the carma, but introduces a potential safety concern as it removes visual indication of the car's automation states.

### Carma version where this issue was discovered

carma 3.3.0+ develop
### Expected behavior
On all passenger cars and trucks:
Essential states:
When the car starts, or carma inactive, all lights should be off. 
When the carma is engaged, the green should on (solid).
When the carma is disengaged, but active, the green flash should on.

In other words, expected state transition:
![image](https://user-images.githubusercontent.com/20613282/97618242-2bf39e00-19f5-11eb-866c-9750d80b88c7.png)
Expected different patterns:
![image](https://user-images.githubusercontent.com/20613282/97618339-46c61280-19f5-11eb-809e-cb9a7767226a.png)

### Actual behavior

Lights are on, but it doesn't reflect the various states of the automation.

### Steps to reproduce the actual behavior

Pull every develop branches and run carma start all.

**Comment by TonysCousin** [comment]
In particular, this issue was brought up by Brian Reed at ATEF, who is concerned about its implications for safety on that track.

**Comment by MishkaMN** [comment]
The issue is resolved by above PRs.
Any further issue is most likely related to lightbar wiring mismatch or reprogrammability
